src/stories/atoms/skeleton.stories.tsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import { Meta, StoryObj } from '@storybook/react';
+
+import { Avatar, Skeleton, Typography } from '../../atoms';
+
+import { Box } from '@mui/material';
+
+const meta = {
+  title: 'Atoms/Skeleton',
+  component: Skeleton,
+  parameters: {
+    layout: 'centered',
+  },
+  tags: ['autodocs'],
+  argTypes: {
+    animation: {
+      control: 'select',
+      options: ['pulse', 'wave', false],
+      description: 'Select an option from the control panel',
+      table: {
+        defaultValue: { summary: "'pulse'" },
+      },
+    },
+    children: {
+      control: { disable: true },
```

Comment:
If  we are not going to let the children be modified, wouldn't it be better to just not show it in the stortybook?
with `table: {disable: true}`
What do you think?